.github/merge-me.yml

```diff
@@ -0,0 +1,3 @@
+reviewTeams:
+- modeling-dev
+- modeling-design
```

Comment:
The documentation on the team names is not 100% clear to me: 

> It deduces the effective teams to account for via the team memberships of existing and requested reviewers

This does mean we don't need the `camunda/` prefix, right? Just want to confirm that `modeling-dev` is enough and we don't need `camunda/modeling-dev`.

Reply:
Yes. Team names are local to the organization.